analyzers/tests/SonarAnalyzer.UnitTest/Rules/NewGuidShouldNotBeUsedTest.cs

```diff
@@ -30,5 +30,9 @@ public class NewGuidShouldNotBeUsedTest
         [TestMethod]
         public void NewGuidShouldNotBeUsed() =>
             Verifier.VerifyAnalyzer(@"TestCases\NewGuidShouldNotBeUsed.cs", new NewGuidShouldNotBeUsed());
+
+        [TestMethod]
+        public void NewGuidShouldNotBeUsed_CSharp9() =>
+            Verifier.VerifyAnalyzerFromCSharp9Library(@"TestCases\NewGuidShouldNotBeUsed.CSharp9.cs", new NewGuidShouldNotBeUsed());
     }
```

Comment:
We prefer to run the C# 9 test cases only on Net context (by using conditional compilation `#if NET`)